pkgs/development/python-modules/python-ironicclient/default.nix

```diff
@@ -25,12 +25,13 @@
 
 buildPythonPackage rec {
   pname = "python-ironicclient";
-  version = "5.10.0";
+  version = "5.10.1";
   pyproject = true;
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-Eg2eakri5+1hPuKyq9+f4dL33il6G4LYLZvoWYfU4LY=";
+  src = fetchgit {
```

Comment:
pypi now has 5.10.1, too. Or we fetch from github. I don't really want to fetch from their crippled gitea.

Reply:
see https://pypi.org/project/python-ironicclient/#files

The tarball has an underscore now.